.github/pull_request_template.md

```diff
@@ -25,12 +25,12 @@
  -->
 
 <!-- Link here the related JIRA issue in case of a bug / feature -->
-### 🏗️ Fixes #(jira-issue-number)
+### 🏗️ Fixes #(jira-issue-number or link)
```

Comment:
Minor nit: put the `#` inside the parenthesis; or even better, substitute it by PROD4POD. As indicated, there's little chance we will actually link automatically jira tickets and GitHub.

Reply:
there's a tool for this and it was in use before, waiting to see if we'll reactivate it.
(but yeah, even if we pay it but ignore and don't use it...there's little chance of everything to get advanced :p)